models/users.js

```diff
@@ -49,10 +49,11 @@ userSchema.pre("save", function (next) {
 
 userSchema.methods.secure = function () {
     // Include virtuals (fullName) in the created object
-    const user = this.toObject({virtuals: true});
+    const user = this.toObject({ virtuals: true });
     //remove sensitive info from user object before sending it back to client
     delete user.password;
     delete user.permissions;
```

Comment:
We can remove the `delete user.permissions` now that it's not part of the schema, right?